packages/checkout-storefront/jest.config.js

```diff
@@ -4,6 +4,7 @@ const { compilerOptions } = requireJSON("./tsconfig.json");
 
 /** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
 module.exports = {
+  testMatch: ["**/**/*.test.ts"],
```

Comment:
If that's a glob shouldn't this also work?

```suggestion
  testMatch: ["**/*.test.ts"],
```